src/assets/colors.tsx

```diff
@@ -0,0 +1,14 @@
+export default {
+  gray: "#f8f8f8",
+  grayRegular: "#e5e5e5",
+  grayMedium: "#cccccc",
+  graySemibold: "#666666",
+  grayBold: "#333333",
+  purpleLight: "#e2d2ff",
+  purpleRegular: "#6e1fff",
+  redRegular: "#f7625a",
+  yellowRegular: "#fdefd1",
+  yellowMedium: "#fce1a8",
+  yellowBold: "#f1af1b",
+  greenRegular: "#0eda83"
+}
```

Comment:
True true. In wonder how others are using this. If the elan data is only being consumed by JS apps, maybe we can just change the JSON file into an ES6 module like this?

/cc @craigspaeth